src/components/ProjectCard.js

```diff
@@ -0,0 +1,34 @@
+import React, { useRef, useEffect, useState } from 'react';
+import Card from 'react-bootstrap/Card';
+import ProjectModal from './ProjectModal';
+
+export default function ProjectCard({ projectData }) {
+  const textRef = useRef();
+  const [overflow, setOverflow] = useState(false);
+  const [readMore, setReadMore] = useState(false);
+
+  useEffect(() => {
+    // Show read more button when the content overflows the card
+    setOverflow(textRef.current.scrollHeight > textRef.current.offsetHeight);
+  }, [projectData.description]);
+
+  return (
+    <Card className="project-card">
+      <Card.Img variant="top" src={projectData.image_url.length !== 0 ? projectData.image_url : "./img/square-pattern.svg"} />
+      <Card.Body>
+        <Card.Title as="h5">{projectData.title}</Card.Title>
+        {/* If there's no tech lead, consider the team empty */}
```

Comment:
I think there are projects where we just only have biz devs I'm pretty sure?

Reply:
Fixed!